README_WINDOWS.md

```diff
@@ -482,145 +348,83 @@ In order to get support for ASIO drivers, follow this directory structure:
-[version 9][dx9sdk] and install it.
-
-*Fine print*: If you install the DirectX SDK and try to build using MinGW, the
-PortAudio build will likely break. Try to avoid the DXSDK detection in that
-case; for example, by editing CMakeLists.txt in the PortAudio root folder.
-DSound support will nevertheless be built when using MinGW libraries. If
-you want to avoid adding in DS-support, set `PA_USE_DS` in the same file to
-`OFF`.
-
-### Configuring and executing the build
-
-The sections below assume that you are comfortable with the command line. If
-not, see the walkthrough below, "Avoiding the command line".
-
-Once everything has been arranged properly, `cmake` must be told:
-
-    - which "generator" to use
-    - the location of the compiler
-    - the location of Qt
-    - the location of SuperCollider's source code
-
-If you're all set, jump to the section below specific to your build tool.
-
-*Note:* You should always use forward slashes for path statements when using
-CMake commands. When creating/modifying environment variables, or interacting
-otherwise directly with Windows command line functionality (`cd`, `rmdir`, ...),
-you should use the proper Windows syntax with backslashes as directory
-delimiters.
-
-*Note:* The SC source given as the last parameter of the `cmake` command. The
-pointer to the source directory is the only element of the cmake command that
-has to be repeated each time cmake is run to configure the build. The other
-options are stored in "CMakeCache.txt" in the build folder for subsequent runs.
-
-#### MinGW (non-MSYS2)
+FFTW does not provide build files for Visual Studio. In the **Developer Command
+Prompt for VS2013** (note that this this is not `cmd.exe`), `cd` to the
+directory where FFTW is installed and, for a **64-bit** build:
 
-The following steps build SuperCollider in a subdirectory of the SC source
-directory called 'build'. You may want to check the path where you installed
-Qt—if it's not `C:\Qt\`, then change these commands accordingly. You should
-`cd` into the SuperCollider source folder to begin.
-
-    SET PATH=C:\Qt\5.5\mingw492_32\bin;C:\Qt\Tools\mingw492_32\bin;%PATH%
-    SET CMAKE_PREFIX_PATH=C:\Qt\5.5\mingw492_32
-    mkdir build
-    cd build
-    cmake -G "MinGW Makefiles" ..
-    cmake --build .
-
-This is the bare-bones, dead-simple way to build SuperCollider with MinGW!
-
-A line-by-line explanation:
-
-    SET PATH=C:\Qt\5.5\mingw492_32\bin;C:\Qt\Tools\mingw492_32\bin;%PATH%
-    SET CMAKE_PREFIX_PATH=C:\Qt\5.5\mingw492_32
-
-Tell `cmake` where to look for the Qt files and MinGW.
-
-    mkdir build
-    cd build
+    lib /machine:x64 /def:libfftw3f-3.def
 
-Create the build directory and move into it.
+For a **32-bit** build:
 
-    cmake -G "MinGW Makefiles" ..
+    lib /def:libfftw3f-3.def
 
-Configure the makefiles to prepare for building.
+The SC build only uses the single precision version of the fftw library
```

Comment:
remove "only" ?

Reply:
I remember that when I was building SC for the first time on Windows, it was a bit confusing because FFTW's site tells you to use the lib command on all three def files, but you in fact just have to do the one. So I'd prefer to leave "only" here.

Actually, I think it might be the use of "version" here that's confusing. You can use both the float and double "versions" without an issue AFAIK. It would probably be most accurate to say "uses the single precision FFTW library". i.e. FFTW project has several libraries, you only need one of them.